src/index.css

```diff
@@ -5,4 +5,5 @@
 
 .main {
   margin: 0;
+  font-family: Inter, Arial, Helvetica, sans-serif;
```

Comment:
where is main applied as a class? do we need to put it on a component (maybe `<App />` in order for it to actually do anything?)

Reply:
I applied it to the `body` tag in `index.html` (in the public folder). I'm not sure how we could remove the body styles otherwise. The font-family we could put in App, but I think we still need to reset the margin on the body.